storybook/src/Storybook/HotReloader.qml

```diff
@@ -0,0 +1,50 @@
+import QtQml 2.14
+import QtQuick 2.14
+
+import Storybook 1.0
+
+QtObject {
+    id: root
+
+    /*required*/ property Loader loader
+    property bool enabled: false
+
+    signal reloaded
+
+    function forceReload() {
+        d.asyncBlocker.when = true
```

Comment:
Why do we need to disable async before we unload the component?

Reply:
I added comment for that in code :)